include/wil/cppwinrt_register_com_server.h

```diff
@@ -48,7 +48,7 @@ void register_com_server(std::vector<wil::unique_com_class_object_cookie>& regis
 {
     DWORD registration{};
     winrt::check_hresult(CoRegisterClassObject(
-        winrt::guid_of<T>(), winrt::make<CppWinRTClassFactory<T>>().get(), CLSCTX_LOCAL_SERVER, REGCLS_MULTIPLEUSE, &registration));
+        __uuidof(T), winrt::make<CppWinRTClassFactory<T>>().get(), CLSCTX_LOCAL_SERVER, REGCLS_MULTIPLEUSE, &registration));

```

Comment:
Please do not remove support for `winrt::guid_of`, thanks!

Reply:
> Please do not remove support for `winrt::guid_of`, thanks!

The WIL `register_com_server` function in question registers COM servers based on CLSID. I do then believe that `__uuidof(T)` would be more appropriate than `winrt::guid_of<T>()` for getting the CLSID for the COM class.

Do you have a concrete example of a COM class where it would be more appropriate to instead use `winrt::guid_of<T>()` for retrieving the CLSID, so that I can better understand your comment?